### ISSUE TITLE
[bug]: sometimes rpc work queue gets exhausted

### ISSUE THREAD

**Issue by chrisguida** [Start9Labs/bitcoind-wrapper#41]
This causes all downstream services to freak out

```
# docker exec -it bitcoind.embassy bitcoin-cli getblockchaininfo
error: couldn't parse reply from server
```

`WARNING: request rejected because http work queue depth exceeded, it can be increased with the -rpcworkqueue= setting`

**Comment by ProofOfKeags** [comment]
Are we misusing it? Is this problem a symptom of a low workqueue or us abusing the rpc?

**Comment by chrisguida** [comment]
I think it's issues with system instability, combined with using the rpc much more heavily

**Comment by chrisguida** [comment]
P2ing this in favor of community-services#9

**Comment by kn0wmad** [comment]
Closing in favor of #63, which in turn should be closed by #69
